src/utils/tableDefinitions.tsx

```diff
@@ -275,7 +284,7 @@ export const loanTypeColumnDefinition = <T extends SideProperty>(
         cell: info => {
             const value = info.getValue();
             return (
-                <div className='mx-auto flex w-[70px] justify-center'>
+                <div className='flex w-[70px] justify-start'>
```

Comment:
I believe there should be some horizontal padding on mobile screens here. It's looking quite squeezy in there:
![Screenshot 2024-07-16 at 1 43 43 PM](https://github.com/user-attachments/assets/a5a876f2-0c60-49c5-9c59-8f4532e3cbb7)

Reply:
It also looks like the Chips are supposed to be shorter (45px)